src/main/fc/fc_main.c

```diff
@@ -149,7 +152,8 @@ void calculateSetpointRate(int axis, int16_t rc) {
 
     if (rcRate > 2.0f) rcRate = rcRate + (RC_RATE_INCREMENTAL * (rcRate - 2.0f));
     rcCommandf = rc / 500.0f;
-    rcDeflection[axis] = ABS(rcCommandf);
+    rcDeflection[axis] = rcCommandf;
+    rcDeflectionAbs[axis] = ABS(rcCommandf);
```

Comment:
To get abs value of float,  you simply clear sign bit.  It's so cheap that caching the result makes little sense